langchain/src/document_loaders/fs/unstructured.ts

```diff
@@ -14,12 +14,21 @@ interface Element {
   };
 }
 
+interface UnstructuredOptions {
+  apiKey?: string;
+}
+
 export class UnstructuredLoader extends BaseDocumentLoader {
-  constructor(public webPath: string, public filePath: string) {
+  constructor(
+    public filePath: string,
```

Comment:
Makes sense to me to add that into `options`. Most users will want to call the default Unstructured API. I'll make that change.

Reply:
Sounds good - any reason to call it `webPath` vs. `apiUrl` also?

Reply:
Main reason was that the web loaders were all using `webPath`, so wanted to keep that consistent. Added links below for two of them, but a bunch of the other web loaders use `webPath` as well. Definitely happy to swap that to `apiUrl` though if that will more clear for users.

- [hn.ts](https://github.com/hwchase17/langchainjs/blob/f97565659c33011ca6a8bc7c5da11aba251911d7/langchain/src/document_loaders/web/hn.ts#L6)
- [college_confidential.ts](https://github.com/hwchase17/langchainjs/blob/f97565659c33011ca6a8bc7c5da11aba251911d7/langchain/src/document_loaders/web/college_confidential.ts#L5)

Reply:
Gotcha - I think `apiUrl` is clearer then if it makes sense on your side. 

I can take it from here, I'll also try to add a temporary shim to avoid breaking people for a little longer, at least until you all start enforcing an API key.

Reply:
Sounds good, thanks! And yep `apiUrl` makes sense to me.